pkg/util/paramtable/component_param.go

```diff
@@ -3073,7 +3073,7 @@ During compaction, the size of segment # of rows is able to exceed segment max #
 	p.ClusteringCompactionMaxTrainSize = ParamItem{
 		Key:          "dataCoord.compaction.clustering.maxTrainSize",
 		Version:      "2.4.0",
-		DefaultValue: "26g",
+		DefaultValue: "25g",
```

Comment:
Magic number is not promising. We can try mmap to help prevent OOM. It is OK for me to keep it for now, we can have create an issue to track the optimization here.